app/navbar.js

```diff
@@ -1,9 +1,6 @@
-// src/components/Navbar.js
 
 "use client";
 
-import React, { useState } from 'react';
```

Comment:
Not quite - `React` is not necessary and so should be removed; `useState` is used below, and so should remain. So you should end up with:
`import { useState } from 'react';`

General tip: Before making a new commit, run the code locally to make sure nothing broke!

Reply:
gotcha! I just fixed it and double checked by running it on my local